test/dub_package/frontend_file.d

```diff
@@ -42,14 +43,19 @@ void main()
     assert(!t.diagnostics.hasWarnings);
 
     t.module_.fullSemantic;
-    auto generated = t.module_.prettyPrint;
+    auto generated = t.module_.prettyPrint.toUnixLineEndings();
 
-    auto expected =q{module foo;
+    version (Windows)
+        enum accumulator = "0.000000";
+    else
+        enum accumulator = "0.00000";
```

Comment:
Why is this necessary?

Reply:
I don't know. The generated code is different on Windows and Posix. Something related to the C standard library that is probably used to format the floating point number?

Reply:
Would be nice to have this as a comment.

Reply:
Sure.

Reply:
Fixed.